src/ccsr/tests/client.rs

```diff
@@ -26,7 +26,7 @@ lazy_static! {
 }
 
 #[tokio::test]
-async fn test_client() -> Result<(), failure::Error> {
+async fn test_client() -> Result<(), anyhow::Error> {
```

Comment:
This might be a controversial opinion, but I'd prefer it if (since you're changing all of these `Result<.., failure::Error>` lines anyway) these modules all did a `use anyhow::Error;` and just returned `Result<.., Error>`, in case we decide to improve error handling again and use module-local errors.

The big advantage would be that the next diff has fewer changes, the smaller the number of lines changed the more likely that semantic changes are noticed.

Reply:
That strikes me as optimizing for the wrong thing. Code is read a lot more than it's written—and changing the error type a particularly uncommon change, no? The fully-qualified signature here is wonderfully descriptive at a glance; it's very obvious which error type is selected without needing to scroll up to the module imports. (When I see a bare `Error`, I think `std::error::Error`.)

This seems like a matter of personal taste more than an issue with a right answer, so I've opted for the status quo and just left the existing style in place everywhere. (There are some modules that `use anyhow::Result` for example.) 

If we want to pick a style and enforce its usage, I'm down for that, but I think we should take a poll and see what folks prefer before committing to one or the other.

Reply:
Yup! I don't think that my opinion is specifically _right_, I just like it. I'm happy with this decision.